app/models/claim.rb

```diff
@@ -257,6 +257,14 @@ def personal_data_removed?
     personal_data_removed_at.present?
   end
 
+  def payrolled?
+    payment.present?
+  end
+
+  def paid?
```

Comment:
The naming of this could be a bit misleading. If the payment date is in the future, it might be _considered_ paid, but not actually paid. I don't think it really matters for our case, but maybe something like `marked_as_paid?` might be better